src/invoices/InvoiceForm/InvoiceForm.js

```diff
@@ -113,6 +113,10 @@ class InvoiceForm extends Component {
     const addressBillTo = get(find(addresses, { id: formValues.billTo }), 'address', '');
     const isEditPostApproval = IS_EDIT_POST_APPROVAL(initialValues.id, initialValues.status);
     const metadata = initialValues.metadata;
+    const users = get(parentResources, 'users.records', []);
+    const approvedByUser = get(users.find(user => user.id === get(initialValues, 'approvedBy')), 'personal', {});
+
+    initialValues.approvedByName = `${get(approvedByUser, 'firstName', '')} ${get(approvedByUser, 'lastName', '')}`;
```

Comment:
Please don't use initialValues for not-form inputs.

Reply:
Fixed